File: src/main/java/nuclearcoder/discordbot/command/misc/CmdPing.java
package nuclearcoder.discordbot.command.misc;

import nuclearcoder.discordbot.BotUtil;
import nuclearcoder.discordbot.NuclearBot;
import nuclearcoder.discordbot.command.Command;
import sx.blah.discord.handle.obj.IMessage;

public class CmdPing implements Command {

    @Override public void execute(NuclearBot bot, IMessage message, String command, String[] args)
    {
        BotUtil.reply(message, "pong!");
    }

}

File: src/main/java/nukebot/command/misc/CmdPing.kt
package nukebot.command.misc

import nukebot.NuclearBot
import nukebot.command.Command
import nukebot.rReply
import sx.blah.discord.handle.obj.IMessage

class CmdPing : Command {

    override fun execute(bot: NuclearBot, message: IMessage, command: String, args: Array<String>) {
        message.rReply("pong!")
    }

}
